addons/vrm/vrm_secondary.gd

```diff
@@ -211,9 +211,12 @@ class SecondaryGizmo:
 	
 	func draw_in_editor(do_draw_spring_bones: bool = false):
 		clear()
-		if secondary_node.get_parent() is VRMTopLevel && secondary_node.get_parent().gizmo_spring_bone:
-			draw_spring_bones(secondary_node.get_parent().gizmo_spring_bone_color)
-			draw_collider_groups()
+		var selected: Array = EditorPlugin.new().get_editor_interface().get_selection().get_selected_nodes()
+		if (secondary_node.get_parent() is VRMTopLevel && selected.has(secondary_node.get_parent())) || selected.has(secondary_node):
+			if do_draw_spring_bones:
+				draw_spring_bones(secondary_node.get_parent().gizmo_spring_bone_color)
+			if secondary_node.get_parent() is VRMTopLevel && secondary_node.get_parent().gizmo_spring_bone:
+				draw_collider_groups()
```

Comment:
Please revert this or change to `&& selected.has`

The code was changed because drawing springbones when the avatar is selected causes heavy lag when testing out animations.